### ISSUE TITLE
Sonoma language popups show up in Taskbar as blank windows

### ISSUE THREAD

**Issue by zihotki** [lawand-dot-io/taskbar#85]
**Describe the bug**

**To Reproduce**
It happens intermittenly and I can't provide any steps to reproduce.

The issue is that sometimes ghost windows appears like it's on screenshot. It happens for multiple applications. The second window has no preview, when I click on it main window opens (the one on the left in the image). If I quit the app and run it again the problem disappears. If I restart the taskbar the problem dissapears.

**Expected behavior**
All existing windows of the application should be visible in the taskbar and no ghost windows should be present

**Screenshots**
<img width="413" alt="image" src="https://github.com/user-attachments/assets/0b30fdee-6e2b-4677-8ed5-0f6dedc4bad9">

**Desktop (please complete the following information):**
 - Taskbar version 1.2.1
 - OS: 14.6.1 (23G93)
 - Chip Apple M2 Pro

**Additional context**

**Comment by lawand-dot-io** [comment]
Hello. Thanks for the bug report. Do you have in mind what apps other than Telegram have produced this bug? Sometimes these issues are app-specific and I need to know which apps are causing the bug in order to fix it. Thanks again

**Comment by zihotki** [comment]
Let me monitor for a short while, take some notes and I'll update the ticket. 

I also noticed that sometimes after a sleep the taskbar becomes empty. But that's probably a separate issue although the solution for both could be the same.

**Comment by lawand-dot-io** [comment]
@zihotki thanks, i would greatly appreciate if you tell me which apps are causing this. I personally only saw firefox doing that.

and about the second bug with the empty taskbar, is there any more info you can give me? e.g. is it only after the laptop sleeping for a long time or perhaps only when many apps are running etc? thanks again for all the help

**Comment by zihotki** [comment]
@lawand-dot-io I think I found at least one cause of ghost windows. I have 2 keyboard inputs/languages configured and when I switch the language MacOs displays small hover on the input. This causes a new window appear in the taskbar. This happens for some software like FreeCad, Telegram, OrcaSlicer, CotEditor, Notes, etc. But for other like VS Code or Edge/Chrome it doesn't happen.

Please see this picture for an example and see that A in blue baloon
![image](https://github.com/user-attachments/assets/39204356-6025-47e0-8fcf-da43c87bf9a7)

If I disable that using steps described in https://stackoverflow.com/questions/77248249/disable-macos-sonoma-text-insertion-point-cursor-caps-lock-indicator namely:

```
defaults write kCFPreferencesAnyApplication TSMLanguageIndicatorEnabled 0
```

 the bug dissapears. And what's the most fascinating and amazing it's that the annoying MacOs switch input lag DISSAPEARED! Wow!

As for the empty toolbar, I'll open a separate issue then when I get more information. If there is a way to enable some useful logging please let me know, I'll be glad to help.

**Comment by lawand-dot-io** [comment]
@zihotki ah, if you mean that there's empty windows only when you change the input language, this is a bug that I am already aware of and have fixed in the development version. You can get that here: https://www.dropbox.com/scl/fi/5nudgn9i8k18hzw7p1hed/Taskbar-1.2.2.zip?rlkey=5bmvsez6e67ayvez3rleerz0h&st=en6xkkux&dl=1

**Comment by lawand-dot-io** [comment]
You can download it and get the fix, or wait for it to be added to next official version. As for the other bug, I like your idea about enabling logging. Let me think about that some more and I'll get back to you

**Comment by lawand-dot-io** [comment]
New ticket opened for blank Taskbar: https://github.com/lawand-dot-io/taskbar/issues/86

**Comment by lawand-dot-io** [comment]
This bug is now fixed in the new release of Taskbar, which is version 1.2.5
